### PR TITLE
Test behavior of media capture from a cross-origin video

### PR DESCRIPTION
**Pull request by dontcallmedom**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/19604)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/289035212)

File: mediacapture-fromelement/cross-origin.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<script src=/resources/testharnessreport.js></script>
+</head>
+<body>
+<video autoplay controls id="output"></video>
+<script>
+
+// Run captureStream() on cross-origin <video> content
+  async_test(function() {
+    const video = document.createElement('video');
+    video.src = location.origin.replace("//", "//www1.") + "/media/test-v-128k-320x240-24fps-8kfr.webm";
```
Comment (yellowdoge) [inline mediacapture-fromelement/cross-origin.html]: test-v-128k-320x240-24fps-8kfr.webm is about 38KB in size. If we don't care about the contents, we could consider using white.webm which is just 10KB, and will perhaps avoid flakes in slow bots.

**Review by yell0wd0g — APPROVED** [review]

**Comment by zcorpan** [comment]
@dontcallmedom can you rebase?

**Comment by alvestrand** [comment]
Closing and reopening to refresh status.

**Pull request merged**